src/openrct2/network/Socket.cpp

```diff
@@ -824,14 +822,14 @@ class UdpSocket final : public IUdpSocket, protected Socket
     }
 
 private:
-    explicit UdpSocket(SOCKET socket, const std::string& hostName)
+    explicit UdpSocket(SOCKET socket, const std::string& hostName) noexcept
```

Comment:
If you're doing a pass over this code, you might as well also modify constructors "sinking" values like this to take `std::string` by value and then `std::move` them to class fields. If rvalues are passed to such constructors, no copying is then performed.

Reply:
Oh I missed those string copies!
I will examine if I can move them or remove `noexcept` if not

Reply:
I removed private `UdpSocket` ctor since it is not being called anywhere.
Other ctors with `const std::string&` now get `std::string` by value and move :)